tests/py3_driver_testers.txt

```diff
@@ -298,6 +298,7 @@ Wouxun_KG-UV6,@KC9HI,11-Dec-2022
 Wouxun_KG-UVD1P,+Wouxun_KG-UV6,11-Dec-2022
 Wouxun_KG-UV8D,Unit tested; needs confirm,24-Dec-2022
 Wouxun_KG-UV8D_Plus,Unit tested; needs confirm,2-Jan-2022
+Wouxun_KG-UV8H,mrtjr1,27-mar-2023
```

Comment:
When you update this file next can you capitalize "mar" like the rest of the things in the file here?

Reply:
Your github id also needs to be prefixed with @. I'll fix both of these things and rebase/squash this for you.

Reply:
adding it to my personal checklist ... Thanks.